### PR TITLE
feat: store entry-dates and support ordering

### PR DESCRIPTION
**Pull request by bram2000**

### CODE REVIEW

File: digital_land/model/entry.py [hunk 1]
```diff
@@ -17,17 +18,28 @@ def __init__(self, data: Item, resource: str, line_num: int = None):
...
         return hash((self.resource, self.line_num, frozenset(self.data.items())))
 
     @classmethod
     def from_facts(
-        cls, entity: str, facts: Set[Fact], resource: str, line_num: int = None
+        cls,
+        entity: str,
+        facts: Set[Fact],
+        resource: str,
+        line_num: int,
```
Comment (ksp37) [inline digital_land/model/entry.py:35]: There's no longer a default for `line_num` - was that intentional?

File: digital_land/repository/entry_repository.py [hunk 2]
```diff
@@ -188,12 +188,13 @@ def _entries_from_rows(self, entity, rows):
...
             line_num = rowdict.pop("__LINE_NUM__")
-            entry_fact[(resource, line_num)].add(
+            entry_date = rowdict.pop("__ENTRY_DATE__")
+            entry_fact[(resource, line_num, entry_date)].add(
                 Fact(entity, rowdict["attribute"], rowdict["value"])
             )
 
         result = {
-            Entry.from_facts(entity, facts, key[0], key[1])
+            Entry.from_facts(entity, facts, key[0], key[1], key[2])
```
Comment (ksp37) [inline digital_land/repository/entry_repository.py:197]: Could use argument expansion (not sure if it will work), i.e. 
`Entry.from_facts(entity, facts, *key)`

File: digital_land/repository/entry_repository.py [hunk 3]
```diff
@@ -124,9 +127,6 @@ def _create_schema(self):
                 entity TEXT,
                 attribute TEXT,
                 value TEXT,
-                entry_date,
-                start_date,
```
Comment (ksp37) [inline digital_land/repository/entry_repository.py:128]: Why were these removed?

**Review by ksp37 — APPROVED** [review]

File: digital_land/model/entry.py (hunk 1 above)
Comment (bram2000) [reply digital_land/model/entry.py:35]: yes - we shouldn't see missing line numbers unless there's a bug.

File: digital_land/repository/entry_repository.py (hunk 2 above)
Comment (bram2000) [reply digital_land/repository/entry_repository.py:197]: nice idea - will try.

File: digital_land/repository/entry_repository.py (hunk 3 above)
Comment (bram2000) [reply digital_land/repository/entry_repository.py:128]: we currently don't have a meaning for them. Can add them again easily if required.
